www/source/docs/blueprints.md

```diff
@@ -0,0 +1,66 @@
+# Blueprints
+Blueprints are default implementations of the build phases and runtime hooks for your application. They also provide some default health check hooks where appropriate to ensure your application is functioning reliably. Customized Blueprints can be created to facilitate re-usability of common patterns in your organization for developing, building, and running your applications.
+
+## Getting Started
+### New Projects
+Simply run `hab plan init` to leverage blueprints. Habitat will look in your application code for known project types and the appropriate `pkg_blueprint`.
+
+If you would like to specify the desired blueprint to use, you can run `hab plan init --blueprint core/java8-maven-tomcat`
+
+The resulting plan will contain all the available callbacks and variables relevant to the blueprint type.
+
+### Existing Plans
+
+To begin using Blueprints, you will need to add the appropriate `pkg_blueprints` 
+
+```bash
+pkg_name="my_java_app"
+....
+blueprint="core/blueprint-java8-maven-tomcat"
```

Comment:
`pkg_blueprint` here - it'll become obvious why we've namespaced all of the plan variables with `pkg_` as soon as blueprints come together, actually 😄. We'll probably suggest that people use a prefix by convention for the feature (`bp_` or something similar) to make sure we don't have conflicts in our namespace